Mind-Benders/Project/index.html

```diff
@@ -0,0 +1,666 @@
+              <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris eleifend, orci sed
+                egestas faucibus, leo sapien laoreet massa, feugia</p>
+            </div>
+          </div>
+
+          <div class="col-md-4 col-sm-12 col-xs-12 py-5 px-5">
+            <div>
+              <img src="./imgs/Dentistry.jpg" alt="Dentistry">
+              <h2>Dentistry</h2>
+              <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris eleifend, orci sed
+                egestas faucibus, leo sapien laoreet massa, feugia</p>
+            </div>
+            <div>
+              <img src="./imgs/Radiology.jpg" alt="Radiology">
+              <h2>Radiology</h2>
+              <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris eleifend, orci sed
+                egestas faucibus, leo sapien laoreet massa, feugia</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="sec4 " >
+      <div class=" conta">
+        <div class="row">
+          <div class="col-md-6 about-jenna animated bounceInLeft large-padding">
+            <h2 class="title text-left"><b>Jenna</b> Berg</h2>
+            <p class="about-title">PROFESSIONAL DENTIST</p>
+            <p id="p1">
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus
+              orci ex, pulvinar nec finibus ut, cursus eget libero. In dictum
+              sem
+            </p>
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus
+              orci ex, pulvinar nec finibus ut, cursus eget libero. In dictum
+              sem non sapien condimentum aliquet. Nulla facilisi. Mauris
+              efficitur dolor dolor, id porta nulla interdum vitae.
+            </p>
+            <div id="btn">
+              <a href="#" target="_self"> View All Doctors</a>
+            </div>
+          </div>
+          <div class="col-md-6">
+            <img src="./images/jenn.jpg" class="pic" alt="jenna">
+          </div>
+        </div>
+      </div>
+      <div class="sec5" id="fqa">
+        <div class="row d-flex align-items-center">
+          <div class="col-md-3 d-flex flex-column justify-content-center align-items-center">
+            <span>53</span>
+            <p>Certified<br />Doctors</p>
+          </div>
+          <div class="col-md-3 d-flex flex-column justify-content-center align-items-center">
+            <span>48</span>
+            <p>Medical<br />Services</p>
+          </div>
+          <div class="col-md-3 d-flex flex-column justify-content-center align-items-center">
+            <span>12</span>
+            <p>Different<br />Affiliates</p>
+          </div>
+          <div class="col-md-3 d-flex flex-column justify-content-center  align-items-center">
+            <span>96</span>
+            <p>Offered<br />Programs</p>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="sec6 ">
```

Comment:
class names would better explain what they do for example a class that changes the background to red is better named "red-background" 😁